src/goblint.ml

```diff
@@ -9,17 +9,11 @@ open Printf
 let main () =
   try
     Maingoblint.reset_stats ();
-    parse_arguments ();
-    check_arguments ();
-    AfterConfig.run ();
-
-    Sys.set_signal (Goblintutil.signal_of_string (get_string "dbg.solver-signal")) Signal_ignore; (* Ignore solver-signal before solving (e.g. MyCFG), otherwise exceptions self-signal the default, which crashes instead of printing backtrace. *)
-
-    Cilfacade.init ();
-
+    Maingoblint.parse_arguments ();
+    Maingoblint.handle_options false;
```

Comment:
`handle options` might be better inside `parse_arguments`. Right now server's `reset_config` forgets to handle after parsing.